src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/UdtTest/UdtTest.cs

```diff
@@ -294,8 +294,6 @@ public void NullTest()
                         {
                             if(e.Message.Contains("Rerun the transaction"))
                                 rerun = true;
```

Comment:
Can you add `{}` for this single statement?

Reply:
Done